packages/console-client/src/kiota-client/api/marketplace/index.ts

```diff
@@ -63,7 +63,7 @@ export interface MarketplaceRequestBuilderGetQueryParameters {
 /**
  * Uri template for the request builder.
  */
-export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace{?includeTenantId*,itemUrns*,perPage*,publishOnMiaDocumentation*,resolveResourcesData*,tenantId*,types*}";
+export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace";
```

Comment:
i'm not sure what is causing this, but i would leave it as was before

```suggestion
export const MarketplaceRequestBuilderUriTemplate = "{+baseurl}/api/marketplace{?includeTenantId*,itemUrns*,perPage*,publishOnMiaDocumentation*,resolveResourcesData*,tenantId*,types*}";
```

Reply:
agree